resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -6,33 +6,47 @@
 from lxml import etree
 import os
 
-parser = argparse.ArgumentParser(description='Create igv session')
+parser = argparse.ArgumentParser(description="Create igv session.")
 
-parser.add_argument("-f", "--files", nargs='+', required=True, help="List of files")
-parser.add_argument("-g", "--genome", type=str, required=True, help="Genome")
+parser.add_argument('-f', '--input_file', required=True, help="File with paths to files for IGV")
 
 args = parser.parse_args()
 
-Global = etree.Element(
-    'Global',
-    genome=args.genome,
-    version='8'
-)
 
-Resources = etree.SubElement(Global, 'Resources')
+def make_xml_tree(input_file):
+    Global = etree.Element(
```

Comment:
This is not a class name - it should be lowercase. Please follow [PEP8 on naming conventions](https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions). 

This also applies to other examples (`Resources`, `Resource`) in this file.